ui/app/components/app/sidebars/sidebar.component.js

```diff
@@ -35,8 +33,6 @@ export default class Sidebar extends Component {
     const { type, sidebarProps = {} } = this.props
     const { transaction = {} } = sidebarProps
     switch (type) {
-      case WALLET_VIEW_SIDEBAR:
-        return <WalletView responsiveDisplayClassname="sidebar-right" />
       case 'customize-gas':
```

Comment:
no need for the sidebar component to know about the wallet view if we are no longer showing it in sidebar. Looks good, makes sense